samples/app-dev/Flow-Design-Concepts/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -1,14 +1,16 @@
-# oneOf-anyOf schema construct
+# allOf-oneOf-anyOf schema construct
 
 
 # Description
 
-In JSON Schema, the keywords anyOf and oneOf are used to specify that a data structure must validate against one or more specified schemas.
+In JSON Schema, the keywords allOf,anyOf,oneOf are used to specify that a data structure must validate against one or more specified schemas.
 The oneOf keyword specifies that the data must validate against exactly one of the schemas listed. If it matches more than one, or none at all, the validation fails.
 The anyOf keyword allows for validation against multiple schemas. If the data validates against at least one of the schemas listed in anyOf, it is considered valid.
+The allOf keyword JSON Schema is used to specify that a JSON document must satisfy all of the specified subschemas. It’s particularly useful for combining multiple schemas into one.
 
 ## Import the sample
-1. Download the sample json file i.e., *oneOf_anyOf_sample.json*.
+1. Download the sample json files i.e., *AllOf_OneOf_AnyOf_Keywords_APISpec_Service.json* and *Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service.json* 
+prior being an service app json and later app json invkoing the service app. Service app created via uploading Open API Specs available in *OpenAPISpecs* folder
```

Comment:
typo in invoking word
Rest triggers in the service app have been configured with Open API Specs 3.0 files. These API Specs are available in the *OpenAPISpecs* folder.